pkgs/tools/admin/awscli2/default.nix

```
@@ -72,6 +72,8 @@ with py.pkgs; buildPythonApplication rec {
     mkdir -p $out/share/zsh/site-functions
     mv $out/bin/aws_zsh_completer.sh $out/share/zsh/site-functions
     rm $out/bin/aws.cmd
+    # Generate completions
+    ${src}/scripts/gen-ac-index --index-location $out/${python3.sitePackages}/awscli/data/ac.index
```

```suggestion
    # Generate completions
    patchShebangs scripts/gen-ac-index
    ./scripts/gen-ac-index --index-location $out/${python3.sitePackages}/awscli/data/ac.index
```

Patching the shebang in the gen-ac-index script, after getting this error:
```
/nix/store/8qagiljbmjs1m6ndchl7b2h9b2vvcx7x-stdenv-linux/setup: /nix/store/1vna73hv36dkbw64l21r0a2hsq8gfxj1-source/scripts/gen-ac-index: /usr/bin/env: bad interpreter: No such file or directory
```

And then replaced `${src}/` with just `./` after getting this error:
```
sed: couldn't open temporary file
```
A little bit of googling made me think that $src is read-only, so the shebang wouldn't be patched. I haven't confirmed that yet, but using just `./` worked for me.

---

Thanks, good catch. I've pushed an update